libs/dao-data/src/Profile.ts

```diff
@@ -66,14 +46,23 @@ export default class Profile {
       index.content?.[
         'kjzl6cwe1jw145cjbeko9kil8g9bxszjhyde21ob8epxuxkaon1izyqsu8wgcic'
       ] ?? null;
-    const doc = await TileDocument.load(client, defId);
-    return doc.content as BasicProfile;
+
+    if (defId) {
+      const doc = await TileDocument.load(client, defId);
+      return doc.content as BasicProfile;
+    } else {
+      return {};
+    }
   }
 
   private async getEns(address: string): Promise<string | null> {
     const provider = new ethers.providers.JsonRpcProvider(
       this.providers['0x1']
     );
-    return await provider.lookupAddress(address);
+    try {
+      return await provider.lookupAddress(address);
+    } catch {
+      return null;
+    }
```

Comment:
So, this is another sort of philosophical thing. Do we hide errors in the library or do we allow clients to handle errors. Personally, I am a fan of the later

Reply:
Yeah worth a discussion.

The spot above was spitting out some bad errors on the Ceramic fetch if `defId` was null in the case the address didn't have a profile. So that seemed like an error in our library not due to what the client is doing.

In the case of profile I kind feel we ought to just return an empty profile + ens if that exists... but I see what you mean.

Reply:
Yea true, in general this can cause big headaches in debugging issues, because we are essentially swallowing the error here.

Reply:
Good point. Adding a ticket to revisit error handling in general for the SDK. 

I've taken a few stabs at it in the Query class and add an error message in the response. But could look more into Throwing as a pattern.

In this case It seems ceramic or our implementation of it was more of a problem. If they throw an error on the missing profile then we kinda have to catch that for our use case. For us a missing profile isn't an error.